src/searchplugin.css

```diff
@@ -22,10 +22,32 @@
     margin-top: 4px;
 }
 
+.nk-search .form-control{
+    width: 250px;
+}
+.nk-search .error{
+    border: 1px solid #ccc;
+    animation: error-blink 2s;
+}
+@keyframes error-blink {
```

Comment:
I never got this to trigger?

Reply:
Didn't it trigger if you wrote whatever text in the input field and hit enter?

Reply:
Nothing seems to happen when I type random garbage in the search field.